pymaker/deployment.py

```diff
@@ -295,9 +301,10 @@ def to_json(self) -> str:
         return self.config.to_json()
 
     @staticmethod
-    def from_network(web3: Web3, network: str):
+    def init(web3: Web3):
```

Comment:
I would have named this `from_node` to maintain consistency with the `from_network` and `from_json` initializers.

Reply:
sounds good, renamed